spec/system/add_search_rspec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Search function', type: :system do
+  before do
+    @ruby_programming_language = create :programming_language, name: 'Ruby'
+    create :programming_language, name: 'Javascript'
+    create :programming_language, name: 'Php'
+
+    @ruby_devloper = create :developer, email: 'phan.nguyen@example.com'
+    @ruby_devloper.developer_programming_languages.create(
+      programming_language: @ruby_programming_language
+    )
+  end
+
+  it 'Being able to search developers who can write Ruby' do
+    visit('/')
+
+    select 'Ruby', from: 'search_developer_form[programming_language]'
+    click_button 'Search'
```

Comment:
`click_on` can be used for both button and link.
I don't know which is better, like always using `click_on` or using `click_button` and `click_link` based on the tag.
But I always use `click_on` personally.

Reply:
@jp-ryuji :+1: